codalab/client/local_bundle_client.py

```diff
@@ -49,10 +46,9 @@
     Group
 )
 from codalab.rest import util as rest_util
+from codalab.rest import bundle as bundle_rest
```

Comment:
Add a comment to this file saying that it will be removed when the rest API is done.